Dynamic data structures/Postfix Calculator/testsForPostfixCalculator.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+bool testCalculate();
```

Comment:
Тут бы тоже комментарии. Кстати. если функция не принимает ничего, пишите void:
```suggestion
bool testCalculate(void);
```